functions.js

```diff
@@ -1,3 +1,25 @@
+// remove once code reviewed
+const testObjectForCodeReview = {
+    name: 'England',
+    code: 'ENG',
+    anagram: 'nldegan'
+}
+/**
+ * function to get display country anagram and country code to html
+ * @param countryObject
```

Comment:
Explain what object is. Maybe describe what properties are needed - ie: anagram & code
eg: '@param {object} countryObject containing anagram & code properties'